src/store/user.js

```diff
@@ -18,6 +18,11 @@ const actions = {
       context.commit('SET_USER', state)
       // force redirect to login page after logout
       router.push({ name: 'login' })
+    }).catch(() => {
+      // reset user to default state
+      context.commit('SET_USER', state)
```

Comment:
assuming logout() returns false; What to do ?
we could pull this out of this promise, because actually the logout action doesn't need any return value...